submissions/ORybka/html-movie-seat-booking/index.html

```diff
@@ -13,7 +13,7 @@
   <main class="main-container">
     <nav class="breadcrumbs">
       <ol class="menu">
-        <li class="menu-item">01 Overview</li>
+        <li class="menu-item previous-menu-item">01 Overview</li>
```

Comment:
If it's an interactive element why you use list element `li` for this?

Reply:
@A-Ostrovnyy I added links as I agree, they will be more suitable for this situation and they will lead to the pages I need. But I left `<li>` as part of the navigation list. Hope this is ok 😃